auth-external-api/src/test/java/uk/gov/di/authentication/external/lambda/UserInfoHandlerTest.java

```diff
@@ -88,6 +85,8 @@ public void setUp() {
         TEST_SUBJECT_USER_INFO.setPhoneNumber("0123456789");
         TEST_SUBJECT_USER_INFO.setClaim(
                 "verified_mfa_method_type", MFAMethodType.AUTH_APP.getValue());
+        TEST_SUBJECT_USER_INFO.setClaim(
+                "current_credential_strength", CredentialTrustLevel.MEDIUM_LEVEL);
```

Comment:
I don't think this is doing anything. When I comment it out, the tests still pass. Could you add some assertion if this is necessary here?

Reply:
It isn't needed here since its all tested in the `userInfoService` . Was following a pattern of an example userInfo. Think only the `email` and `phone number` is needed. Can remove both the `verified_mfa_method_type` and `current_credential_strength` claims.

Reply:
It isn't used anywhere, but we do expect it to be on the response. The `UserInfoServiceTest` tests that we conditionally add claims, but this file tests the full handler's response. So we can, for example, just add 
```
assertTrue(
                response.getBody()
                        .contains(String.format("\"current_credential_strength\":\"MEDIUM_LEVEL\"")));
```
to verify this is on the response on the first test in this file. Or perhaps better still would be testing the whole response body as a JSON object. What do you reckon?

Reply:
I think I do want it in so therefore we should assert something on it. We mock the userInfoService so all we are testing is that the object gets passed through into the response. We don't check the object apart from email and phone number. But I think I prefer that over nothing. Will make the change.